1091. Shortest Path in Binary Matrix/solution.py

```diff
@@ -0,0 +1,34 @@
+class Solution:
+    # Given a grid, returns the shortest clear path from the 
+    # top-left corner to the bottom-right corner. A clear path is one 
+    # that traverses only 0s and can go diagonally. If there is none,
+    # returns -1.
+    def shortestPathBinaryMatrix(self, grid: List[List[int]]) -> int:
+        # Uses a BFS traversal to get the shortest path from the
+        # top-left to the bottom-right so needs a queue. Entries are
+        # in the form of [row, col].
+
+        # Define a version of the grid but for the length of the 
```

Comment:
"Define a version" doesn't make much sense. Also don't know what you mean "for the length of shortest cleaar path" since a grid has a length and a width. Better to say: "Create another grid with the same dimensions to keep track of whether a cell has been visited or not"

Reply:
Actually reading further, I can't figure out if you defined a new grid or are editing the existing grid in place.